### ISSUE TITLE
WARNING: Config values are not defined in any .rc file: opt

### ISSUE THREAD

**Issue by npuichigo** [deepmind/sonnet#49]
It seems `--config=opt` doesn't make sense.

**Comment by Shuolongbj** [comment]
FYI.
#25 
#45

**Comment by Shuolongbj** [comment]
$python rnn_shakespeare_test.py

WARNING:tensorflow:From /Users/Ru/anaconda/lib/python3.6/site-packages/sonnet/examples/rnn_shakespeare.py:308: reuse_vars (from sonnet.python.modules.experimental) is deprecated and will be removed after 2017-08-01.
Instructions for updating:

The @snt.experimental.reuse_vars decorator has been moved to @snt.reuse_variables. Please change to use the new location.

/Users/Ru/anaconda/lib/python3.6/site-packages/tensorflow/python/util/tf_inspect.py:45: DeprecationWarning: inspect.getargspec() is deprecated, use inspect.signature() or inspect.getfullargspec()

  if d.decorator_argspec is not None), _inspect.getargspec(target))

WARNING:tensorflow:The `skip_connections` argument will be deprecated. Please use snt.SkipConnectionCore instead.

..
----------------------------------------------------------------------
Ran 2 tests in 569.119s

OK
$ pip show sonnet
Name: sonnet
Version: 1.5
Summary: Sonnet is a library for building neural networks in TensorFlow.
Home-page: http://www.github.com/deepmind/sonnet/
Author: DeepMind
Author-email: sonnet-dev-os@google.com
License: Apache 2.0
Location: /Users/Ru/anaconda/lib/python3.6/site-packages
Requires:
$ pip show tensorflow
Name: tensorflow
Version: 1.2.1
Summary: TensorFlow helps the tensors flow
Home-page: http://tensorflow.org/
Author: Google Inc.
Author-email: opensource@google.com
License: Apache 2.0
Location: /Users/Ru/anaconda/lib/python3.6/site-packages
Requires: html5lib, bleach, backports.weakref, protobuf, wheel, six, werkzeug, markdown, numpy

**Comment by Shuolongbj** [comment]
~/dnc$ python train.py --memory_size=64 --num_bits=8 --max_length=3

INFO:tensorflow:2899: Avg training loss 18.487717.

Observations:
+- - - 1 - - - - - - - -+
+- - - - - - - - - - - -+
+- - 1 1 - - - - - - - -+
+- - 1 - - - - - - - - -+
+- - - 1 - - - - - - - -+
+- 1 1 - - - - - - - - -+
+- - 1 1 - - - - - - - -+
+- - 1 1 - - - - - - - -+
+1 - - - - - - - - - - -+
+- - - - 1 - - - - - - -+

Targets:
+- - - - - - - 1 - - - -+
+- - - - - - - - - - - -+
+- - - - - - 1 1 - - - -+
+- - - - - - 1 - - - - -+
+- - - - - - - 1 - - - -+
+- - - - - 1 1 - - - - -+
+- - - - - - 1 1 - - - -+
+- - - - - - 1 1 - - - -+
+- - - - - - - - 1 - - -+

Model Output:
+- - - - - - - - - - - -+
+- - - - - - - - - - - -+
+- - - - - 1 1 1 1 - - -+
+- - - - - - - - - - - -+
+- - - - - 1 1 - - - - -+
+- - - - - 1 1 1 1 - - -+
+- - - - - 1 1 1 - - - -+
+- - - - - 1 1 1 1 - - -+
+- - - - - - - - - - - -+

**Comment by TobyPDE** [comment]
Can you try to run the build with the additional flag `--bazelrc=tensorflow/tools/bazel.rc`?

**Comment by diegolascasas** [comment]
The warning should be fixed now, thanks for pointing that out.
